src/platform/linux.c

```diff
@@ -49,12 +53,21 @@ typedef enum { DT_NONE, DT_KDIALOG, DT_ZENITY } dialog_type;
 void platform_get_exe_path(utf8 *outPath, size_t outSize)
 {
 	char exePath[MAX_PATH];
+#ifdef __LINUX__
 	ssize_t bytesRead;
 	bytesRead = readlink("/proc/self/exe", exePath, MAX_PATH);
 	if (bytesRead == -1) {
 		log_fatal("failed to read /proc/self/exe");
 	}
 	exePath[bytesRead - 1] = '\0';
+#else
```

Comment:
If this is not applicable to Open/NetBSD, you should guard this section with explicit check for FreeBSD